desktop/app/main.js

```diff
@@ -35,6 +34,9 @@ const menubarLoadingIconPath = resolveAssets('../react-native/react/images/menub
 const mb = menubar({
   index: `file://${resolveAssets('./renderer/launcher.html')}?src=${hotPath('launcher.bundle.js')}`,
   width: 320,
+  height: 364 + 10, // size plus gap to deal with deadzone
+  transparent: true,
+  resizable: false,
```

Comment:
Don't let them resize this. Transparent allows us to do the arrow (and workaround the deadzone issue, i'll make a fix for the trackers and pinetry to use this short term)